### ISSUE TITLE
Sign up for newsletter

### ISSUE THREAD

**Issue by wtphilip** [stormkit-io/www-stormkit-io#50]
Currently, as visitor of the homepage of Stormkit, there is no way to get information about updates except for social media. 

There are two ways to enable this:
1. Pop-up window
2. Element on website 

I am personally not a fan of large pop-up windows, because they have a negative impact on user experience, and dedicating a line to sign up to a newsletter is wasted space that does not let the visitor focus on the content. 

Maybe the best option is a small pop-up window after the visitor scrolled and has been on the landing page for 5 seconds that is visible at the bottom left or right that asks something like:
"Give us your e-mail and we'll spam you... NOT. We will inform you about significant changes."

**Comment by svedova** [comment]
Thanks for submitting the issue @wtphilip  :) I'm assigning the labels. 

For the first time visitors we already have the GDPR banner popping up. Perhaps a second banner could be exhaustive. What if we display an input in the footer area of every page and interested users can subscribe to the newsletter?

**Comment by wtphilip** [comment]
I agree. Having two large banners popping up the same time is very bad. I was rather thinking of 2 line pop up at the bottom, when users scroll to the bottom. If they do not get that far, it does not show up. On the other hand, it would also work as link in the footer, but that comes across less proactive.

**Comment by svedova** [comment]
Since in both versions the user would need to scroll to bottom, we can create an input that is visible enough in the top of the footer and try that way. If we see there is not much interaction we can always iterate :)
